test/create-open.test.js

```diff
@@ -138,7 +138,7 @@ Object.keys(testAPIs).forEach(API => {
           assert.equal(manifest.name, 'second')
           assert.equal(manifest.type, 'feed')
           assert.notEqual(manifest.accessController, null)
-          assert.equal(manifest.accessController.indexOf('/ipfs'), 0)
+          assert.ok(manifest.accessController.indexOf('/ipfs') === 0 || manifest.accessController.indexOf('\\ipfs') === 0)
```

Comment:
It'd be great if you could add an explicit test for testing `\` -> `/` conversion and to provide an example how the addresses and paths look like on Windows platforms. Not sure how exactly to test it, though. What do you think?